app/javascript/components/widget-display.jsx

```diff
@@ -147,4 +151,18 @@ DisconnectedDisplay.defaultProps = {
   error: false,
 };
 
+export const WidgetDisabledDisplay = () => (
+  <Wrapper>
+    <FullPanel>
+      <WidgetDisabledMessage />
+    </FullPanel>
+    <SidePanel
```

Comment:
Do we need to display the SidePanel? Can you include a screenshot of what the "No widget enabled" message looks like?

Reply:
We paired on this and determined that the SidePanel should be included :+1:. Please do add a screenshot to the PR description once @cbarber lets you know how to get an empty array returned for the widgets!

Reply:
@allisonlucas  sure.